src/java.desktop/share/classes/java/awt/FlowLayout.java

```diff
@@ -51,31 +51,38 @@
  * <li>{@link #TRAILING TRAILING}
  * </ul>
  * <p>
- * For example, the following picture shows an applet using the flow
+ * For example, the following picture shows a window using the flow
  * layout manager (its default layout manager) to position three buttons:
  * <p>
- * <img src="doc-files/FlowLayout-1.gif"
+ * <img src="doc-files/FlowLayout-1.png"
  * ALT="Graphic of Layout for Three Buttons"
  * style="margin: 7px 10px;">
```

Comment:
I'd drop this `style` attribute. It indents the image by 10 pixels to the right so that it doesn't align to the left edge of the surrounding text.

I see no value in increasing the vertical margins by 7px. Each paragraph has already 14px top/bottom margins.